lib/dry/cli.rb

```diff
@@ -97,6 +97,10 @@ def call(arguments: ARGV, out: $stdout, err: $stderr)
     # @api private
     def perform_command(arguments)
       command, args = parse(kommand, arguments, [])
+
+      command.instance_variable_set(:@stderr, err)
```

Comment:
The idea hear is to not have to change the signature of `Command#initialize` causing a breaking change

Reply:
How about initializing a pair of `StringIO` in the command, and then

```ruby
def perform_command(arguments)
  command, args = parse(kommand, arguments, [])
  command.call(**args)
  IO.copy_stream(command.out, out)
  IO.copy_stream(command.err, err)
end
```

This would make testing the stdout and stderr of commands much simpler.

Reply:
The intention is to utilize the out/err passed to the CLI in the first place. dry-cil uses them to print help and errors and "inline" commands. We **should** also allow the commands to use the user specified out/err